src/services/RejectDialogService.ts

```diff
@@ -19,19 +26,19 @@ module.exports = async (req, res) => {
         .message("Resource not found")
         .send();
 
-    const index = dialogue.approvals.indexOf(userId);
+    const index = dialogue.approvals.indexOf(userId as Schema.Types.ObjectId);
```

Comment:
Need to say `userId as Schema.Types.ObjectId` every time? is there a way to use it just once and reuse it everywhere else?